content/en/security/code_security/secret_scanning/github_actions.md

```diff
@@ -0,0 +1,73 @@
+---
+description: Use Datadog and GitHub to detect Secrets in code in a CI pipeline.
+is_beta: true
+title: Secrets Scanning and GitHub Actions
+---
+## Overview
+
+Run a [Datadog Secrets Scanning][1] job in your GitHub Action workflows. This action wraps the [Datadog Static Analyzer][8],
+that scans for secrets, invokes it against your codebase, and uploads the results to Datadog.
+
+## Workflow
+
+Create a file in `.github/workflows` to run a Datadog Static Code Analysis job.
```

Comment:
we call it a Datadog secrets scanning job and now a Datadog Static Code Analysis job. I think the former is clearer to the purpose of the topic. Let me know if that's a mistake.

Reply:
You updated and that seems totally fine for me! Thanks for checking and fixing it!